core/lib/qcbor/sub.mk

```diff
@@ -0,0 +1,5 @@
+global-incdirs-y += .
+cflags-remove-y += -mgeneral-regs-only
```

Comment:
Looks like you need:
```
cflags-y += -Wno-declaration-after-statement
cflags-y += -Wno-redundant-decls
```

Please add a commit to expand CI with `CFG_REMOTE_ATTESTATION_PTA=y` so we can see any build error/warning (similar to 266c83c88).

Reply:
Added cflags to ignore warnings.
In addition, ci is also modified.